servicenow-itsm.sgnl.yaml

```diff
@@ -0,0 +1,689 @@
+description: "ServiceNow ITSM as a System of Record"
+address: "{{Input Required: Instance-Name}}.service-now.com"
+defaultSyncFrequency: HOURLY
+defaultSyncMinInterval: 1
+defaultApiCallFrequency: SECONDLY
+defaultApiCallMinInterval: 1
+type: "ServiceNow-1.0.0"
+# Example Config:
+# {"version": 1, "config": {"requestTimeout": "10s", "servicenow": {}}}
+adapterConfig: "CnsKCSJ2ZXJzaW9uIjogMSwKCSJjb25maWciOiB7CgkJInJlcXVlc3RUaW1lb3V0IjogIjEwcyIsCgkJInNlcnZpY2Vub3ciOiB7fQoJfQp9Cg=="
+# All auth mechanisms specified below must be supported by the specified Adapter.
+auth:
+  - basic:
+      username: "{{Input Required: Username}}"
+      password: "{{Input Required: Password}}"
+entities:
+  User:
+    displayName: User
+    externalId: sys_user
+    description: User Entity in ServiceNow
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 999
```

Comment:
@joewelsh would love your feedback on Disney's config here. @shash-sgnl for context, we found that the SNOW API was timing out at this page size. I'd recommend something between 250 and 400 (for all entities) based on my experience testing at Disney. But Joe can provide first-hand feedback here too.

Reply:
Hmm, not entirely sure why we set it that high.  Last I recall was a conversation with Romain regarding this, but could be wrong.  I'd stick with the recommended 250-400.

Reply:
Set pagesize for all entities to 350..